javascript/FieldToggleDescription.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Enable toggling (show/hide) of the field's description.
+ */
+
+(function ($) {
+
+    $.entwine('ss', function ($) {
+
+        $('.toggle-description').entwine({
```

Comment:
This classname is not documented anywhere is it? You'd need to include some kind of instructions for users.

I think the rule is, is this documentation in a place where someone not explicitly looking for it, but who could benefit from it, able to find it? Somewhere in docs/en/02_Developer_Guides/03_Forms/?